AK/String.h

```diff
@@ -101,6 +101,7 @@ class String : public Detail::StringBase {
     ErrorOr<String> to_uppercase(Optional<StringView> const& locale = {}) const;
     ErrorOr<String> to_titlecase(Optional<StringView> const& locale = {}, TrailingCodePointTransformation trailing_code_point_transformation = TrailingCodePointTransformation::Lowercase) const;
     ErrorOr<String> to_casefold() const;
+    ErrorOr<String> to_fullwidth() const;
```

Comment:
This should really be 2 commits - first commit for "AK+LibUnicode" to implement `to_fullwdith`, a second commit for LibWeb to make use of it in text-transform